README.md

```diff
@@ -0,0 +1,30 @@
+<img height='200' src="https://user-images.githubusercontent.com/37406965/51083189-d5dc3a80-173b-11e9-8ca0-28015e0893ac.png" align="left" hspace="1" vspace="1">
+
+# PPI-Vision Android Application for MifosX
+
+Vision PPI is a computer vision and machine learning based app to help in filling out the PPI Survey. There are two broad aspects to this project - The android app which provides the interface to the field officer for conducting the survey and the machine learning models which will be used to analyze the images captured.
+## Screenshots
+
+<p>
+  <img src="https://user-images.githubusercontent.com/38397893/72104939-c1ba2180-332c-11ea-8671-c8191c3927ec.png" />
```

Comment:
Combining all 4 screenshots into one single image makes it look a bit congested. Could you please make a 2*2 table and place the 4 images separately on it. That ought to make the readme look neater.